stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -127,8 +127,8 @@ iszero(D::Diagonal) = all(iszero, D.diag)
 isone(D::Diagonal) = all(isone, D.diag)
 isdiag(D::Diagonal) = all(isdiag, D.diag)
 isdiag(D::Diagonal{<:Number}) = true
-istriu(D::Diagonal) = true
-istril(D::Diagonal) = true
+istriu(A::Diagonal) = all(istriu, A.diag)
+istril(A::Diagonal) = all(istril, A.diag)
```

Comment:
Why change `D` to `A`? In that file, there are 93 occurences of `D::Diagonal` vs. 12 occurences of `A::Diagonal`, and it used to be `D` in those lines?

Reply:
ok, i am correcting this!